lib/backend/k8s/resources/node_conversion.go

```diff
@@ -70,6 +70,11 @@ func mergeCalicoK8sNode(calicoNode *model.Node, k8sNode *kapiv1.Node) (*kapiv1.N
 	if calicoNode.BGPIPv4Net != nil {
 		subnet, _ := calicoNode.BGPIPv4Net.Mask.Size()
 		ipCidr := fmt.Sprintf("%s/%d", calicoNode.BGPIPv4Addr.String(), subnet)
+
+		// If the Annotations map is nil, initialize it.
```

Comment:
This turned out to be causing a panic in the new tests.

I had thought this would always be initialized, but seems like perhaps not.